### ISSUE TITLE
Update some basic CRUD examples

### ISSUE THREAD

**Issue by mfp22** [state-adapt/state-adapt#34]
https://github.com/state-adapt/state-adapt/discussions/33

**Comment by mfp22** [comment]
https://github.com/mfp22/2023-fidelity-workshop/tree/state-adapt

**Comment by timsar2** [comment]
> https://github.com/mfp22/2023-fidelity-workshop/tree/state-adapt (If you run it locally, do `npm install --legacy-peer-deps`. There's something going on with `jest-preset-angular`. Please submit a PR if you know how to fix it.)

it has circular dependency:
>  NX   Could not execute command because the task graph has a circular dependency

   earnings-app:build:production --> auth:build:production --> shared-state-auth:build:production --> auth:build:production

**Comment by mfp22** [comment]
I remember seeing this at one point. I need to fix up this example. I didn't create it originally. But I think I only saw this when creating a prod build maybe? Are you able to run it with `npm start`, or is this what you're getting for that?

**Comment by timsar2** [comment]
> I remember seeing this at one point. I need to fix up this example. I didn't create it originally. But I think I only saw this when creating a prod build maybe? Are you able to run it with `npm start`, or is this what you're getting for that?

Yes it worked with npm start but not working with production build.

**Comment by mfp22** [comment]
New docs examples coming soon, and more example apps
